gradle-consistent-versions-idea-plugin/src/main/java/com/palantir/gradle/versions/intellij/RepositoryExplorer.java

```diff
@@ -103,21 +113,48 @@ private Set<GroupPartOrPackageName> fetchFoldersFromContent(String contents) {
     }
 
     private Set<DependencyVersion> parseVersionsFromContent(String content) {
-        Set<DependencyVersion> versions = new HashSet<>();
         try {
             XmlMapper xmlMapper = new XmlMapper();
+            xmlMapper.registerModule(new GuavaModule());
```

Comment:
Can avoid GuavaModule here by setting `@Value.Style(jdkOnly = true)` on the immutables type. This is actually good practice; internally we have [a library](https://pl.ntr/2oV) for this that sets some good defaults.